conn.go

```diff
@@ -38,6 +38,14 @@ type PreSharedKeyCache interface {
 
 type PSKMapCache map[string]PreSharedKey
 
+// A CookieHandler does two things:
+// - generates cookies that are sent to the client in the HelloRetryRequest
+// - validates cookies sent by the client in a ClientHello
+type CookieHandler interface {
+	Generate(*Conn) (*CookieExtension, error)
+	Validate(*Conn, *CookieExtension) bool
```

Comment:
Can you add a comment on what Validate's returns are?

Reply:
I thought this is covered by the comment for the interface. Which additional information would you like here?